docs/HOWTO-bn.md

```diff
@@ -1,34 +1,36 @@
-# How-To at a glance
+# তথ্য এক নজরে
 
 <div align="right" markdown="1">
 
 *[অন্য ভাষায় এটা পড়ুন](README.md#translations)*
 
 </div>
 
-**`Free-Programming-Books` রিপজিটরি তে স্বাগতম!**
+**`Free-Programming-Books`-এ স্বাগতম!**
 
-আমরা নবাগত কন্ট্রিবিউটরস্ দের স্বাগতম জানাই; এমনকি যারা GitHub এ প্রথম কোন Pull Request (PR) তৈরি করেছেন। যদি আপনি তাদের একজন হয়ে থাকেন তাহলে নিচের রিসোর্স গুলো আপনার কাজে লাগতে পারেঃ
+আমরা নতুন যোগদানকারীদের স্বাগত জানাই; এমনকি যারা তাদের প্রথম পুল রিকোয়েস্ট (PR) টি গিটহাবে প্রেরণা নিয়ে তৈরি করছেন। আপনি যদি একজন তাদের মধ্যে থাকেন, তবে নিম্নলিখিত সম্পদগুলি সাহায্য করতে পারে:
```

Comment:
This sounds a little bit google-translatey. More fluent and usable Bangla translate would be like this:
> আমরা নতুন যোগদানকারীদর স্বাগত জানাই; এমনকি যারা তাদের প্রথম পুল রিকোয়েস্ট (PR) গিটহাব এ তৈরি করছেন, তাদেরকেও।  আপনি যদি তাদের মধ্যে থাকেন, তবে নিম্নলিখিত রিসোর্সগুলি সাহায্য করতে পারে: